spring-boot-modules/spring-boot-3-testcontainers/src/main/java/com/baeldung/testcontainers/jdbc/Hobbit.java

```diff
@@ -0,0 +1,39 @@
+package com.baeldung.testcontainers.jdbc;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity(name = "hobbits")
+public class Hobbit {
+
+	@Id
```

Comment:
looks like we are out of format (indents) 

make sure you are using our formatter: 

https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/intellij

https://github.com/Baeldung/dev-settings/tree/main/javaCodeStyle/eclipse

Reply:
You are right, I was using the wrong formatter.

Hopefully, we'll be able to automate this formatting check soon :)